packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,313 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { transparentize } from 'polished';
+
+/**
+ * @children required
+ */
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
+  testId?: string;
+  /*
+   * Custom number for specific breakpoint responsive styling
+   */
+  breakpoint?: number;
+  /*
+   * Determines the Stepper view based on a minimum breakpoint number.
+   */
+  breakpointStyle?: BreakPointStyle;
+  /*
+   * Sets a custom label for the Step count # of #.
+   */
+  stepDescriptionLabel?: string;
+  /*
+   * Sets a custom label for the completed Step.
+   */
+  completedStepDescription?: string;
+  /*
+   * Current step value
+   */
+  currentStep?: number;
+  /**
+   * @internal
+   */
+  hasBreakpoint?: boolean;
+  /*
+   * Inverse styling
+   */
+  isInverse?: boolean;
+  /*
+   * Hides the Step labels
+   */
+  hideLabels?: boolean;
+  /*
+   * Changes from Step labels on each Step to one label at a time.
+   */
+  summaryView?: boolean;
+  /**
+   * @internal
+   */
+  stepStatus?: StepStatus;
+  /**
+   * @internal
+   */
+  theme?: ThemeInterface;
+}
+
+export enum BreakPointStyle {
```

Comment:
I'm also confused by this name, what does the breakpoint have to do with the labels/summary?

Reply:
Changed to `StepperLayout`.